src/app/components/TransactionOverview/TransactionOverview.tsx

```diff
@@ -24,19 +24,22 @@ export default function TransactionOverview({
       </H2>
 
       <ul>
-        {transactions.map((transaction: TransactionProps) => (
-          <Li key={transaction.id}>
-            <CategoryTitle>{transaction.category}</CategoryTitle>
+        {transactions &&
+          transactions.map((transaction: TransactionProps) => (
+            <Li key={transaction.id}>
+              <CategoryTitle>{transaction.category}</CategoryTitle>
 
-            <span>
-              {' '}
-              €{Math.abs(transaction.amount)} ,-
-              <Icon>
-                <TrashIcon onClick={() => deleteTransaction(transaction.id)} />
-              </Icon>
-            </span>
-          </Li>
-        ))}
+              <span>
+                {' '}
+                {/* €{transaction.amount ? Math.abs(transaction.amount): null} ,- */}
```

Comment:
Why is this commented out?

Reply:
I have a question in this line...so that I do not forget